pkg/service_sync/service_sync.go

```diff
@@ -113,6 +113,7 @@ func (c *ServiceSync) localDefinitionsUpdated(definitions map[string]types.Servi
 			EnableTls:                original.EnableTls,
 			TlsCredentials:           original.TlsCredentials,
 			PublishNotReadyAddresses: original.PublishNotReadyAddresses,
+			Namespace:                original.Namespace,
```

Comment:
My initial feeling is that the namespace should not be included in the advertised information, but should remain visible only within the originating site. Do you need service-sync to propagate this information?

Reply:
Indeed. Giving it a second thought, this information is not relevant from the remote site perspective